subworkflows/local/snv_annotation/tests/main.nf.test.snap

```diff
@@ -17,6 +17,19 @@
             ],
             "## Using API version 110, DB version 110",
             [
+                "## gnomADg version v3.1.2",
+                "## assembly version GRCh38.p14",
+                "## 1000genomes version phase3",
+                "## gnomADe version r2.1.1",
+                "## COSMIC version 97",
+                "## sift version 6.2.1",
+                "## polyphen version 2.2.3",
+                "## ClinVar version 202301",
+                "## gencode version GENCODE 44",
+                "## regbuild version 1.0",
+                "## genebuild version 2014-07",
+                "## dbSNP version 154",
+                "## HGMD-PUBLIC version 20204",
```

Comment:
Not sure why this shows up here, before #244 is merged. maybe it's because I updated the test cache with plugins. The config should not be changed in this PR.